test/verify/check-connection

```diff
@@ -317,13 +317,13 @@ class TestConnection(MachineCase):
         else:
             cockpit_ws = "/usr/libexec/cockpit-ws"
 
-        # start ws with --stdio-bridge, and a bridge as user
+        # start ws with --local-session, and a bridge as user
         script = '''#!/bin/bash -eu
 coproc env G_MESSAGES_DEBUG=all cockpit-bridge 2>/tmp/bridge.log
 # wait for bridge to fork and start up
 while ! test -s /tmp/bridge.log; do sleep 1; done
 sleep 5
-%s -p 9999 -a 127.0.0.90 --no-tls --stdio-bridge <&${COPROC[0]} >&${COPROC[1]} 2>/tmp/ws.log
+%s -p 9999 -a 127.0.0.90 --no-tls --local-session=- <&${COPROC[0]} >&${COPROC[1]} 2>/tmp/ws.log
```

Comment:
Needs another test case for a `--local-session=/usr/bin/cockpit-bridge` argument.